libkbfs/prefetcher.go

```diff
@@ -561,7 +592,7 @@ func (p *blockPrefetcher) run(testSyncCh <-chan struct{}) {
 			//
 			// `numBlocks` now represents only the number of blocks to add
 			// to the tree from `pre` to its roots, inclusive.
-			numBlocks, isTail, err := p.handlePrefetch(pre)
+			numBlocks, isTail, err := p.handlePrefetch(pre, isParentNew)
```

Comment:
Looks like `isParentNew` is always the same as `isPrefetchWaiting`.  It's a little weird to have a variable about a "parent" above, since there's no notion of parents/children in this function, we're just processing a single block.